.travis.yml

```diff
@@ -10,7 +11,10 @@ install:
   - nuget install NUnit.Runners -Version 2.6.4 -OutputDirectory ./testrunner
 script:
   - xbuild /p:Configuration=Release ./OptimizelySDK.Travis.sln
-  - mono ./testrunner/NUnit.Runners.2.6.4/tools/nunit-console.exe ./OptimizelySDK.Tests/bin/Release/OptimizelySDK.Tests.dll 
+  - mono ./testrunner/NUnit.Runners.2.6.4/tools/nunit-console.exe ./OptimizelySDK.Tests/bin/Release/OptimizelySDK.Tests.dll
+after_success:
+  - pip install --user awscli
```

Comment:
nit: add a comment here saying where we are doing.

I like that we do the install here instead of in the install step because this really is only needed if it succeeds. 

Also, shouldn't we only upload if it is the master branch?